src/app/shared/services/action-state.service.ts

```diff
@@ -13,6 +13,7 @@ export class ActionStateService {
   private readonly actionsSubjects: Map<string, BehaviorSubject<Action[]>> = new Map()
   private readonly eventSubscriptions: EventSubscription[]
   private readonly logger: Logger
+  private readonly noRundownId: string = 'noRundownIdAllActions'
```

Comment:
This looks like it behaves like a constant. If that's true then it shouldn't be created as a instance variable, but instead as a constant:
```
const ALL_ACTIONS: string = 'noRundownIdAllActions'

export class StateService {
  ...

  private resetActionsSubject(): void {
    ...
    this.logger.debug(`Resetting actions with id: ${rundownId ?? ALL_ACTIONS}`)
  } 
}
```

Reply:
Done